package.json

```diff
@@ -1,52 +1,53 @@
-{
-  "name": "ubiquibot",
-  "version": "0.0.2",
-  "description": "Enables Ubiquity DevPool support in your repo.",
-  "author": "Ubiquity DAO",
-  "license": "ISC",
-  "homepage": "https://github.com/ubiquity/bounty-bot.git",
-  "keywords": [
-    "probot",
-    "github",
-    "probot-app"
-  ],
-  "scripts": {
-    "build:local": "tsc",
-    "build:serverless": "ncc build src/index.ts -o dist && cp ./dist/index.js ./bin",
-    "clean": "rimraf ./dist ./lib ./node_modules",
-    "start:serverless": "probot run ./dist/index.js",
-    "start:local": "probot run ./lib/main.js",
-    "start:local:watch": "nodemon --exec 'yarn start:local'",
-    "prettier": "prettier --write src",
-    "prettier:check": "prettier -c src/**/*.ts",
-    "_commit:watch": "nodemon --exec 'git add . && git commit -am \"update(ci)\" && git push'",
-    "prepare": "husky install"
-  },
-  "dependencies": {
-    "@probot/adapter-github-actions": "^3.1.3",
-    "@sinclair/typebox": "^0.25.9",
-    "@vercel/ncc": "^0.34.0",
-    "ajv": "^8.11.2",
-    "ajv-formats": "^2.1.1",
-    "husky": "^8.0.2",
-    "nodemon": "^2.0.19",
-    "prettier": "^2.7.1",
-    "probot": "^12.2.4",
-    "telegraf": "^4.11.2"
-  },
-  "devDependencies": {
-    "@types/jest": "^28.1.0",
-    "@types/node": "^18.0.0",
-    "@types/source-map-support": "^0.5.6",
-    "jest": "^26.6.3",
-    "nock": "^13.0.5",
-    "rimraf": "3.0.2",
-    "smee-client": "^1.2.2",
-    "source-map-support": "^0.5.21",
-    "ts-jest": "^26.4.4",
-    "typescript": "^4.8.4"
-  },
-  "engines": {
-    "node": ">= 10.13.0"
-  }
-}
+{

```

Comment:
lol

Reply:
....

Reply:
One More Time :/

Reply:
@0xcodercrane do you have any idea what's happening here? @ManyRios reverted the file and then added the dependencies correctly, presumably, using `yarn add` yet Git is considering the entire file to have changed? 

@ManyRios did you change the permissions on the file or something? I have no idea why this is happening.

Reply:
No I do not touch permissions and I used yarn for the dependencies.